conf/mime.types

```diff
@@ -1,7 +1,8 @@
 
 types {
     text/html                                        html htm shtml;
-    text/css                                         css;
+    text/css                                         csv;
```

Comment:
```suggestion
    text/css                                         css;
```

Reply:
God I'm really looking forward to the end of the week